ITE/9/README.adoc

```diff
@@ -45,7 +45,33 @@ The link:https://github.com/in-toto/attestation/[in-toto Attestation Framework]
 
 When discussing a new attestation type, what is really being defined is a new type of link:https://github.com/in-toto/attestation/tree/main/spec#predicate[predicate]. The developer(s) of this new predicate must create a formal document that specifies the predicate type, and discusses several other related aspects. The purpose and prerequisites for the new attestation type MUST be defined. The document should also include a model that describes how the various components of the predicate map to supply chain semantics, juxtaposed with detailed use cases. These use cases SHOULD include example metadata that clearly highlight how the proposed predicate addresses the shortcoming described by them. Including these aspects will aid readers and future adopters understand the why and how of a particular attestation type.
 
-The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` following the rules defined by the attestation framework. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/main/spec/v1.0-draft#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
+The specification's front matter MUST unambiguously declare a unique predicate
+type, a version, and a unique, shorter predicate name following the rules
+defined by the attestation framework. The predicate type along with the version
+maps to the `predicateType` field in the `Statement` layer of an in-toto
+attestation.
+
+The short predicate name is used as the name of any files related to the
+predicate specification (specification document, directories, protobuf
+definition). The name must not include version information and is expected to
+only be used as an unauthenticated hint in other contexts such as a media type
+to indicate the predicate stored in an attestation.
+
+For example, the in-toto link predicate has the predicate type
+`https://in-toto.io/attestation/link/VERSION`.  Its short name is just
+`link`.  Consequently, the specification file must be named `link.md` while the
+protobuf definition must be named `link.proto`. A media type for an attestation
+file that contains a link predicate may also use `link` to indicate the
+predicate, for example `application/vnd.in-toto.link+dsse`.
+
+Further, the specification MUST include a detailed schema of the fields that
```

Comment:
Are they required to include an implementation of the specification parsing / handling code as well?

Reply:
Typically the author of the predicate spec is expected to provide a protobuf definition for the predicate, which in turn provides APIs for predicates thanks to code gen.

note that this isn't a change introduced in this PR, it's only in the diff because of line wrapping / new paragraph.

Reply:
The ITE may also not be the best place to specify other requirements for a new predicate's spec, as they may change over time. I suggest we scope the ITE to the high level workflow (how to submit a new predicate, who reviews it) and the predicate specification's contents. The attestation repo's [docs](https://github.com/in-toto/attestation/blob/main/docs/new_predicate_guidelines.md) can handle other requirements.